src/pre_process/m_patches.f90

```diff
@@ -940,6 +942,13 @@ subroutine s_2D_analytical(patch_id, patch_id_fp, q_prim_vf) ! -----------------
                     !END IF
                     ! ================================================================================
 
+                    ! Taylor Green Vortex===============================================
```

Comment:
This shouldn't go here... it masks any other possible analytic patches someone might want to use and forces one to recompile to comment out the code to change it. Instead do something similar to the isentropic vortex case.

Reply:
Ok, I will work on this.